.babelrc

```diff
@@ -1,19 +1,14 @@
 {
-  "presets": ["es2015", "stage-0", "react"],
-  "env": {
-    "development": {
-      "plugins": [
-        ["react-transform", {
-          "transforms": [{
-            "transform": "react-transform-hmr",
-            "imports": ["react"],
-            "locals": ["module"]
-          }, {
-            "transform": "react-transform-catch-errors",
-            "imports": ["react", "redbox-react"]
-          }]
-        }]
-      ]
-    }
-  }
+  "presets": [
```

Comment:
Couldn't you just use `babel-presets-latest` instead of `es2015`/`es2016`/`es2017`?